COMMUNICATION.md

```diff
@@ -1,5 +1,17 @@
-# Communication Channels
+# Join Our Community
 
-TBD -- finalize all communication channels
+We rely on the contribution and participation of our members. Whether you're here to contribute code, share knowledge, or simply support the continued development of our open-source projects, our community spaces welcome everyone ready to make a positive impact.
 
-- Mailing lists are in `CONTACTS.yaml`
+## Engage, Learn, and Share
+
+- [**GitHub Discussions**](https://www.commonhaus.org/feed/index.rss): Share ideas, ask questions, and help shape the evolution of our collective effort.
```

Comment:
Shouldn't there be a direct link to the Discussions page (in order to post a topic) rather than just consuming the feed?

Reply:
That's a bad link that is.